tests/fips/fips_setup.pm

```diff
@@ -56,7 +56,7 @@ sub install_fips {
     zypper_call("in -t pattern fips") if (is_sle('<=15-SP3') || get_var("FIPS_ENV_MODE"));
     # crypto-policies script reports Cannot handle transactional systems.
     if (is_sle_micro) {
-        is_sle_micro('<6.0') ? trup_call("pkg install -t pattern fips") : trup_call("setup-fips");
+        is_sle_micro('<6.0') ? trup_call("pkg install -t pattern microos-fips") : trup_call("setup-fips");
```

Comment:
We can remove the outer `if` above.

Also, why use the ternary operator for flow control? =/

Reply:
I was hoping that someone will point this out as I didn't want to make that change on my own